### PR TITLE
Fix broken link in admin/add-users/strong-password.md

### PR DESCRIPTION
**Pull request by HarryAllen1**

### CODE REVIEW

File: microsoft-365/admin/add-users/strong-password.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ This article explains how to turn off strong password requirements for your user
 
 ## Before you begin
 
-This article is for people who manage password policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. [What's an admin account?](Overview of the Microsoft 365 admin center](../admin-overview/admin-center-overview.md) You must be an [global admin or password administrator](about-admin-roles.md) to perform these steps.
+This article is for people who manage password policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. [What's an admin account?](../admin-overview/admin-center-overview.md#overview-of-the-microsoft-365-admin-center) You must be an [global admin or password administrator](about-admin-roles.md) to perform these steps.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/strong-password.md]:
```suggestion
This article is for people who manage password policy for a business, school, or nonprofit. To complete these steps, you need to sign in with your Microsoft 365 admin account. [What's an admin account?](../admin-overview/admin-center-overview.md#overview-of-the-microsoft-365-admin-center) You must be a [global admin or password administrator](about-admin-roles.md) to perform these steps.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MajesticString, not related, but kindly also commit the additional suggested change to correct the typo. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**